modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1045,10 +1045,9 @@ class Edit_User extends \NDB_Form
             && (!empty($values['Password_hash']) || !empty($values['__Confirm']))
         ) {
             try {
-                $password = new \Password($values['Password_hash']);
```

Comment:
This line is necessary. please add it back or else the password won't get validated.

Reply:
already back